packages/lib/src/vis/tiles/Tile.tsx

```diff
@@ -29,6 +29,7 @@ function Tile(props: Props) {
         {...colorMapProps}
         magFilter={magFilter}
         size={{ width, height }}
+        scale={new Vector3(1, 1, 1)}
```

Comment:
Override default `HeatmapMesh` scale.

Reply:
That won't be needed if you implement https://github.com/silx-kit/h5web/pull/1052/files#r841521948